### ISSUE TITLE
Pre-capillary pulmonary hypertension

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6493]
**Preferred term label:**
Pre-capillary pulmonary hypertension

**Synonyms**

**Definition (free text, please give PubMed ID)**
Pre-capillary pulmonary hypertension is a haemodynamic condition characterised by elevated mean pulmonary artery pressure (mPAP > 20 mmHg) and pulmonary vascular resistance (PVR ≥ 3 WU)  accompanied by normal pulmonary artery wedge pressure (PAWP ≤ 15 mmHg) PMID: 30545968

**Parent term (use hpo.jax.org/app)**
Elevated pulmonary artery pressure HP:0004890 (Consider a change to elevated mean pulmonary artery pressure?)
(I would suggest moving pulmonary arterial hypertension HP:0002092 as a child term of pre-capillary pulmonary hypertension)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Idiopathic and/or familial pulmonary arterial hypertension (IFPAH) ORPHA:422, OMIM: 178600,   OMIM:265400,  OMIM: 615342,  OMIM:615343, OMIM:  615344
Idiopathic pulmonary arterial hypertension ORPHA:275766
Hereditary pulmonary arterial hypertension ORPHA:275777
Pulmonary arterial hypertension associated with connective tissue diseases ORPHA:275798
Pulmonary arterial hypertension associated with HIV infection ORPHA:275808 
Pulmonary arterial hypertension associated with portal hypertension ORPHA:275813  
Pulmonary arterial hypertension associated with congenital heart diseases ORPHA:275803
Pulmonary arterial hypertension associated with schistosomiasis ORPHA:275823
Pulmonary arterial hypertension with overt features of venous/capillaries (PVOD/PCH) involvement ORPHA:431353
Pulmonary hypertension due to lung diseases and/or hypoxia ORPHA:275837
Chronic thromboembolic pulmonary hypertension ORPHA:70591
Pulmonary hypertension with unclear multifactorial mechanisms ORPHA:275844
Pulmonary hypertension associated with chronic hemolytic anaemia ORPHA:275828

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik  -- In order to emphasize that this is a phenotypic feature, I wonder if we could use the name 
Increased precapillary pulmonary blood pressure

**Comment by eswietlik** [comment]
I agree.

**Comment by pnrobinson** [comment]
added
